macros/apply-tags/apply_tags.sql

```diff
@@ -0,0 +1,37 @@
+{% macro apply_tags(resource) -%}
+  {{ return(adapter.dispatch('apply_tags', 'dbt_tags')(resource)) }}
+{%- endmacro %}
```

Comment:
Should we rename this macro to a better name, suggested as `apply_column_tags_query`?

Reply:
Can you help to add yml documentation file for this macro?